src/pages/Animal/Feed/index.tsx

```diff
@@ -24,11 +25,13 @@ const FeedPets = (): JSX.Element => {
   }, [navigation]);
 
   const fetchPets = (): void => {
-    animalAPI.getAll()
-      .then((result) => {
-        const data = result.docs.map((doc) => ({ id: uuidv4(), ...(doc.data()) }));
-        setFetchedPets(data);
-      });
+    const user = userAPI.currentUserDocument();
+    const query = user ? animalAPI.getNotOwnedByUser(user) : animalAPI.getAll();
```

Comment:
Conforme a conversamos ontem, essa página foi reimplementada usando o componente de InfiniteScroll do André, portanto, é bem provavel que essas mudanças seja descartadas. O mesmo vale para a página de meus pets.

Reply:
I'll keep the API so that you can replace when merging with Infinite Scroll PR